Cargo.toml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+resolver = "2"
+members = [
+    "up-streamer",
+    "examples/simple-example",
+]
+
+[workspace.package]
+rust-version = "1.66.1"
```

Comment:
is this aligned (and verified) with all of your upstream dependencies?

Reply:
Will check